src/vcRender.cpp

```diff
@@ -687,13 +687,15 @@ void vcRenderTerrain(vcState *pProgramState, vcRenderContext *pRenderContext)
 
     int currentZoom = 21;
 
-    double farPlane = pProgramState->settings.camera.farPlane;
+    // This technique won't work with ECEF
+    // This 'value' was trial and errored. 
+    double visibleFarPlane = 1000.0 + udAbs(localCamPos.z) * 150.0;
```

Comment:
Similar to above `constexpr` comment, if both `1000.0` and `150.0` can be given meaningful names, it would be nice to seem a `constexpr double name = ...;` for each. If it's decided to go down this path.